pkg/operators/socketenricher/socketenricher.go

```diff
@@ -157,7 +193,15 @@ func (i *SocketEnricherInstance) EnrichEvent(ev any) error {
 }
 
 func (s *SocketEnricher) GlobalParams() api.Params {
-	return nil
+	return api.Params{
+		{
+			Key:          fieldsParam,
+			Title:        "Socket enricher fields",
+			Description:  "List of fields and their sizes to the enabled on the socket enricher. It uses the field0=size,field1=size,... format",
+			DefaultValue: "cwd=512,exepath=512",
```

Comment:
That's a breaking change in the CLI flags since gadget param `--paths` will no longer be enough to enable the feature.  Are you planning to explain this in the documentation?

Reply:
> With the current default value, cwd and exepath will be enabled by default.

That's right. The default value of the `socket-enricher-fields` params enables the cwd and exepath fields with a length of 512 bytes, it's the same behavior before this PR.

> I was afraid that this PR will force users to understand the socket enricher to be able to use gadgets like trace_dns. 

If the system admin (the ones that can set the `socket-enricher-fields` param) disables those fields, them will be empty even if the user passes --paths. I think we can mention it somehow on the docs of each gadget. I'll do it once this PR is more mature.